src/smart_qq_bot/config.py

```diff
@@ -13,7 +13,7 @@
 
 DEFAULT_PLUGIN_CONFIG = "config/plugin.json"
 
-COOKIE_FILE = 'cookie/cookie.data'
+COOKIE_FILE = '../cookie/cookie.data'
```

Comment:
Nice change! It makes the cookie directory a constant:)

But for the path constants, we have two more choices here:
1. the current working directory(If we run run.py, we could get the directory what we expected)
2. An abs directory with "os.path.abspath(os.path.dirname(__file__))"
So "../" is not so fluent as we expected : ), it is the up level of the current working directory.
Could you please change this setting to either of abs paht or working directory?